config/initializers/filter_parameter_logging.rb

```
@@ -1,10 +1,8 @@
-# frozen_string_literal: true
-
 # Be sure to restart your server when you modify this file.
 
 # Configure parameters to be partially matched (e.g. passw matches password) and filtered from the log file.
 # Use this to limit dissemination of sensitive information.
 # See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
-Rails.application.config.filter_parameters += %i[
-  passw email secret token _key crypt salt certificate otp ssn cvv cvc
+Rails.application.config.filter_parameters += [
+  :passw, :email, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn, :cvv, :cvc
 ]
```

RuboCop (1.85.1)

[convention] Use `%i` or `%I` for an array of symbols.

(Style/SymbolArray)